examples/pyscannerbit_hds_interface.py

```diff
@@ -0,0 +1,101 @@
+import high_dimensional_sampling as hds
+from high_dimensional_sampling import results
+import numpy as np
+
+### PS stuff ###
+import pyscannerbit.scan as sb
+
+from mpi4py import MPI
+rank = MPI.COMM_WORLD.Get_rank()
+size = MPI.COMM_WORLD.Get_size()
+
+new_scans = True
+###
+
+
+class HDS_PS_INTERFACE(hds.Procedure):
+    def __init__(self):
+        self.store_parameters = []
+        self.reset()
+    
+    def __call__(self, function):
+        # Get ranges of the test function. The 0.001 moves the minima 0.001 up
+        # and the maxima 0.001 down, in order to make use the sampling is not
+        # by accident moving outside of the test function range.
+        ranges = function.get_ranges(0.01)
+        ranges = np.array(ranges).tolist()
+        
+        simple = function.get_simple_interface()
+        # Can this be done without specifiying dimensionality? HDS and PS are looking for max/min respectively.
+        neg_f = lambda x,y: -simple(x,y)
```

Comment:
Solution to issue #12: Replace with

if dimensions == 1:
            neg_f = lambda a: -simple(a)
        elif dimensions == 2:
            neg_f = lambda a,b: -simple(a,b)
        elif dimensions == 3:
            neg_f = lambda a,b,c: -simple(a,b,c)
        elif dimensions == 4:
etc.

Reply:
See #12